src/lockfree.h

```diff
@@ -26,37 +26,46 @@
 
 #include <boost/lockfree/stack.hpp>
 
+template <std::size_t TSize, size_t CAPACITY>
+struct LockfreeFreeList
+{
+	using FreeList = boost::lockfree::stack<void*, boost::lockfree::capacity<CAPACITY>>;
+	static FreeList& get()
+	{
+		static FreeList freeList;
+		return freeList;
+	}
+};
+
```

Comment:
Why not using a static variable instead of wrapping with a class?

Reply:
this is C++11 and variable templates were supported in C++14